dvc/progress.py

```diff
@@ -182,9 +181,29 @@ def set_size(self, size):
             size = self.fs.info(self.path_info)["size"]
         self.progress_bar.total = size
         self.progress_bar.refresh()
+        super().set_size(size)
 
     def relative_update(self, inc=1):
         self.progress_bar.update(inc)
+        super().relative_update(inc)
 
     def absolute_update(self, value):
         self.progress_bar.update_to(value)
+        super().absolute_update(value)
+
+
+def tdqm_or_callback_wrapped(
```

Comment:
This codebase is a bit weird, as we need to support both tqdm and `FsspecCallback` for now.